src/clj/puppetlabs/services/jruby/jruby_metrics_core.clj

```diff
@@ -366,7 +367,7 @@
    free-instances-fn :- IFn
    registry :- MetricRegistry]
   {:num-jrubies (metrics/register registry (metrics/host-metric-name hostname "jruby.num-jrubies")
-                  (metrics/gauge max-active-instances))
+                  (if jruby-puppet-core/multithreaded? (metrics/gauge 1) (metrics/gauge max-active-instances)))
```

Comment:
It might make more sense to change the caller here to pass in the max pool size instead of `max-active-instances`, so we don't have to do a conditional. That happens here. https://github.com/puppetlabs/puppetserver/blob/master/src/clj/puppetlabs/services/jruby/jruby_metrics_service.clj#L51-L58

Reply:
not entirely sure what you mean, but I'm guessing it doesn't matter anymore since we decided to overload. this can just go back to `max-active-instances`.